Gaijinizer patch/Map349.txt

```diff
@@ -59,8 +59,8 @@ What is...!!?#GT
 な、なんだ…！
 何が起こったんだ…！？
 > CONTEXT: Map349/events/5/pages/1/24/Dialogue/kazuya-c
-...!!?
+...?!!
 
-What is...!
-What happened...!?#GT
+W-What...?!
+What's happened...?!#Ron
 > END STRING
```

Comment:
Is this happened, or happening?

Reply:
I can't remember the exact context, but this is "happened". As in "What('s) happened here?"

Reply:
Just checked, it's when he turns into Kid Kazuya. We'll want `What happened...?!` here.